core/src/main/java/software/amazon/awssdk/retry/conditions/RetryCondition.java

```diff
@@ -13,13 +13,19 @@
  * permissions and limitations under the License.
  */
 
-package software.amazon.awssdk.retry.v2;
+package software.amazon.awssdk.retry.conditions;
+
+import software.amazon.awssdk.retry.RetryPolicyContext;
+import software.amazon.awssdk.retry.SdkDefaultRetrySettings;
 
-/**
- * Super interface for {@link RetryPolicy} used to define when a request should be retried.
- */
 public interface RetryCondition {
 
+    RetryCondition DEFAULT = new OrRetryCondition(new RetryOnStatusCodeCondition(SdkDefaultRetrySettings.RETRYABLE_STATUS_CODES),
```

Comment:
Thoughts on making these default methods like Predicate does?

https://docs.oracle.com/javase/8/docs/api/java/util/function/Predicate.html#or-java.util.function.Predicate-

Reply:
Fine with adding and/or default methods for any given two retry conditions but not sure it makes sense in the line you are highlighting here. 

Think it makes sense to do one condition and another condition. Don't think it makes sense to do a condition and a list of others. You can do retryCondition and RetryCondition and RetryCondition but then you are needlessly adding layers.